packages/node_modules/nav-frontend-etiketter/src/index.tsx

```diff
@@ -36,19 +28,16 @@ export interface EtikettBaseProps extends  EtikettProps {
     type: 'suksess' | 'info' | 'advarsel' | 'fokus';
 }
 
-class EtikettBase extends React.Component<EtikettBaseProps> {
-    render() {
-        const { type, className, typo = 'normaltekst', children, ...props } = this.props;
-
-        return (
-            <div className={cls(type, className)} {...props}>
-                <TypografiBase tag="span" type={typo}>{children}</TypografiBase>
-            </div>
-        );
-    }
+function EtikettBase(props: EtikettBaseProps): JSX.Element {
```

Comment:
Vet ikke om omskriving til funksjonkomponent kan.
Var tidligere problemer med at autogenerering av eksempler og props da ikke viste riktig. 
Så det bør testes imho.

Reply:
Stemmer 👍 Disse må forbli klasse-komponenter for at `react-docgen-typescript` skal funke.